systest/Makefile

```
@@ -36,13 +36,15 @@ ifeq ($(configname),$(test_job_name))
 	run_deps = config
 endif
 
-command := tests -test.v -test.count=$(count) -test.timeout=0 -test.run=$(test_name) -clusters=$(clusters) \
--level=$(level) -configname=$(configname)
+command := tests -test.v -test.count=$(count) -test.timeout=60m -test.run=$(test_name) -test.parallel=$(clusters) \
+					 -clusters=$(clusters) -level=$(level) -configname=$(configname)
 
 ifeq ($(failfast),true)
 	command := $(command) -test.failfast
 endif
```

NIT: we don't need the if I believe and can just pass the value to the CLI

```suggestion
command := tests -test.v -test.count=$(count) -test.timeout=60m -test.run=$(test_name) -test.parallel=$(clusters) \
	-test.failfast=$(failfast) -clusters=$(clusters) -level=$(level) -configname=$(configname)
```